src/ImageRunModal.jsx

```diff
@@ -118,7 +131,7 @@ class PublishPortsForm extends React.Component {
         );
     }
 }
-PublishPortsForm.propTypes = {
+DynamicListForm.propTypes = {
     onChange: PropTypes.func.isRequired,
     id: PropTypes.string.isRequired,
```

Comment:
Let's add the optional `formclass` and the required `itemcomponent` and `default`.  *if* a component has propTypes, it should be complete.

Reply:
oki